examples/asr/other/QuartzNetModel.ipynb

```diff
@@ -0,0 +1,454 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NeMo Models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[NeMo W 2020-05-28 17:08:17 audio_preprocessing:56] Could not import torchaudio. Some features might not work.\n",
+      "[NeMo W 2020-05-28 17:08:17 audio_preprocessing:61] Unable to import APEX. Mixed precision and distributed training will not work.\n"
```

Comment:
I generally cleanup notebooks output before submitting, do we want to keep outputs in repo to showcase what is down there without actually running?

Reply:
good point. forgot to do it. cleaned up now